### DISCUSSION TITLE
Clustertruck

### DISCUSSION THREAD

**Discussion by 13gkd** [skyline-emu/skyline#1398] [General]
![Screenshot_2022-05-28-17-32-03-733_skyline emu](https://user-images.githubusercontent.com/103353628/170824701-8b9a0381-ca7f-44e3-b555-85a6ab430b1c.jpg)
![Screenshot_2022-05-28-17-32-12-358_skyline emu](https://user-images.githubusercontent.com/103353628/170824707-ef378603-16ac-48b9-bd6b-b5c259de0bc9.jpg)
Only showing this
![Screenshot_2022-05-28-17-32-23-008_skyline emu](https://user-images.githubusercontent.com/103353628/170824739-821cb462-1f46-4e56-a83d-6a7e5856dfe3.jpg)
And last showing black screen and no sound
![Screenshot_2022-05-28-17-33-06-545_skyline emu](https://user-images.githubusercontent.com/103353628/170824764-9f1dc238-d8c8-4bb8-96cc-ac87c0d11730.jpg)

**Comment by Joemama45** [comment]
Weird, cluster truck runs perfectly normal for me 
![Screenshot_20220529-191049_Skyline](https://user-images.githubusercontent.com/97562949/170895014-b6c3bdc7-f8d2-4185-b5ad-2bd60399d8af.jpg)
